src/guide.jl

```diff
@@ -345,7 +345,7 @@ function render_continuous_color_key(colors::Dict,
          (context(),
           line([[(0, 1 - y), (swatch_width, 1 - y)] for y in values(colors)]),
           linewidth(theme.grid_line_width),
-          stroke(color("white"))),
+          stroke(parse(Color, "white"))),
```

Comment:
This, and similar changes below, are not strictly necessary since I added `color` to Compose. I can drop this part if you prefer.